src/EnergyPlus/EconomicTariff.cc

```diff
@@ -4581,9 +4640,14 @@ namespace EconomicTariff {
 		numMins = 0;
 		MinTariffIndex.dimension( numTariff, 0 );
 		for ( iTariff = 1; iTariff <= numTariff; ++iTariff ) {
-			//determine if this is meter related to electricity
+			// TODO: is this needed?
+			// Assume it's not a water meter (already handled in the constructor, since set to 0)
+			tariff( iTariff ).kindWaterMtr = kindMeterNotWater;
+
+			//determine if this is meter related to electricity, or water
 			if ( tariff( iTariff ).reportMeterIndx != 0 ) {
```

Comment:
Oh - I see now, the proposed changes include a commented out block in GetInput.  So depending on how this turns out, it might make sense to move it up there.